ecmascript/transforms/src/typescript/strip.rs

```diff
@@ -16,17 +16,19 @@ type EnumValues = FxHashMap<Id, TsLit>;
 
 #[derive(Debug, Serialize, Deserialize)]
 #[non_exhaustive]
+// TODO(nayeemrmn): The name should be `ImportsNotUsedAsValues`. Rename as a
+// breaking change.
 pub enum ImportNotUsedAsValues {
     #[serde(rename = "remove")]
     Remove,
     #[serde(rename = "preserve")]
     Preserve,
 }
 
-/// This value defaults to `Preserve`
+/// This value defaults to `Remove`
 impl Default for ImportNotUsedAsValues {
     fn default() -> Self {
-        Self::Preserve
+        Self::Remove
```

Comment:
Is this safe?
Even if the default of tsc is remove, `swc` preserved those.

Reply:
I meant `breaking change` for rust side.
I'll not make a breaking change for js apis. Sorry if you are confused.

Reply:
Reverted 👍

Reply:
Wait @kdy1 the effect of `ImportNotUsedAsValues` is completely different from before. Defaulting it to remove would actually align with the current behaviour. Do you mind if I change the default to "remove"? It would be safe after all.

Reply:
No, I don't mind it.
I think it's safe anyway as they are not used.

Reply:
Sorry, I was confused about it.